src/main/java/com/manardenza/console/VisualUserMenu.java

```diff
@@ -0,0 +1,168 @@
+package com.manardenza.console;
+
+import com.manardenza.entity.AbstractObject;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class VisualUserMenu {
+
+    public static void outputSplitLine() {
+        String line = "****************************************";
+        System.out.println(line);
+    }
+
+    public static void outputDataInConsole(String message) {
+        System.out.println(message);
+    }
+
+    public static <E extends AbstractObject> void outputListInConsole(Set<Map.Entry<String, List<E>>> set) {
+        int key = 1;
+        for (Map.Entry<String, List<E>> entry : set) {
+            System.out.print(key++);
+            System.out.println(" " + entry.getKey());
+            for (int item = 0; item < entry.getValue().size(); item++) {
+                System.out.print("\t" + (item + 1));
+                System.out.println(" " + entry.getValue().get(item));
+            }
+        }
+    }
+
+    static void showUserMenu(String massage) {
+        String[] mainMenu = new String[2];
+        mainMenu[0] = "Login";
+        mainMenu[1] = "Exit";
+
+        String[] serviceMenu = new String[6];
+        serviceMenu[0] = "Search hotel by name city";
+        serviceMenu[1] = "Search by hotel name";
+        serviceMenu[2] = "Search room";
+        serviceMenu[3] = "Reservation order";
+        serviceMenu[4] = "Cancel reservation";
+        serviceMenu[5] = "Exit";
+
+        String[] bookingMenu = new String[2];
+        bookingMenu[0] = "Make a reservation";
+        bookingMenu[1] = "No reserve";
+
+        if (massage.equals("Main menu")) {
+            sortMenuItems(mainMenu);
+        }
+        if (massage.equals("Service menu")) {
+            sortMenuItems(serviceMenu);
+        }
+        if (massage.equals("Booking menu")) {
+            sortMenuItems(bookingMenu);
+        }
+
+        System.out.println();
+    }
+
+    private static void sortMenuItems(String[] array) {
+        for (int i = 0; i < array.length; i++) {
+            System.out.println(String.format("%d. %s", i + 1, array[i]));
+        }
+    }
+
+    static Integer readerMenuFromConsole(String action) throws IOException {
+        boolean flag = true;
+        int number = 0;
+        outputDataInConsole(action);
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        while (flag) {
+            try {
+                number = Integer.parseInt(reader.readLine());
+            } catch (NumberFormatException ex) {
+                outputDataInConsole("Enter number: ");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Enter number: " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)